public-api/src/integration/java/org/pmiops/workbench/publicapi/DataBrowserControllerIntegrationTest.java

```diff
@@ -0,0 +1,106 @@
+package org.pmiops.workbench.publicapi;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.assertj.core.util.Strings;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.publicapi.client.ApiClient;
+import org.pmiops.workbench.publicapi.client.api.DataBrowserApi;
+import org.pmiops.workbench.publicapi.client.model.AchillesResult;
+import org.pmiops.workbench.publicapi.client.model.Analysis;
+import org.pmiops.workbench.publicapi.client.model.Concept;
+import org.pmiops.workbench.publicapi.client.model.ConceptAnalysis;
+import org.pmiops.workbench.publicapi.client.model.ConceptListResponse;
+import org.pmiops.workbench.publicapi.client.model.Domain;
+import org.pmiops.workbench.publicapi.client.model.DomainInfosAndSurveyModulesResponse;
+import org.pmiops.workbench.publicapi.client.model.SearchConceptsRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * Integration smoke tests for the Data Browser API - intended to run against a live instances of
+ * the Data Browser API. These tests should be relatively unspecific, as they must run across
+ * different CDR versions.
+ */
+@RunWith(SpringRunner.class)
+public class DataBrowserControllerIntegrationTest {
+
+  public static String DB_API_BASE_PATH = "DB_API_BASE_PATH";
+
+  @TestConfiguration
+  static class Configuration {
+    @Bean
+    DataBrowserApi client() {
+      DataBrowserApi api = new DataBrowserApi();
+      String basePath = System.getenv(DB_API_BASE_PATH);
+      if (Strings.isNullOrEmpty(basePath)) {
+        throw new RuntimeException("Required env var '" + DB_API_BASE_PATH + "' not defined");
+      }
+      api.setApiClient(new ApiClient().setBasePath(basePath));
+      return api;
+    }
+  }
+
+  @Autowired
+  DataBrowserApi api;
+
+  @Test
+  public void testDomainTotals() throws Exception {
+    DomainInfosAndSurveyModulesResponse resp = api.getDomainTotals();
+    assertThat(resp.getDomainInfos()).isNotEmpty();
+    assertThat(resp.getSurveyModules()).isNotEmpty();
+  }
+
+  @Test
+  public void testGenderCount() throws Exception {
+    Analysis resp = api.getGenderAnalysis();
+    // TODO: These are empty - is that expected??
+    // assertThat(resp.getDistResults()).isNotEmpty();
+    // assertThat(resp.getResults()).isNotEmpty();
+  }
+
+  @Test
+  public void testParticipantCount() throws Exception {
+    AchillesResult resp = api.getParticipantCount();
+    assertThat(resp.getCountValue()).isGreaterThan(0L);
+    // TODO: Source count was 0 in test env - expected for synthetic data?
```

Comment:
Yes this would be 0 as source count value was just not being generated at all. 
https://github.com/all-of-us/data-browser/blob/master/public-api/db-cdr/generate-cdr/run-achilles-queries.sh#L56